effect-handlers/wasmtime
# Disable most CI jobs

This completely disables the following CI jobs, which I assume are the ones we don't need.

 - cargo_deny
 - cargo_vet
 - checks_winarm64
 - fuzz_targets
 - test_wasi_nn
 - test_wasi_crypto
 - bench
 - meta_deterministic_check
 - verify-publish
 - miri
 - build-preview1-component-adapter
